tests/test_task_details.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+# noinspection PyRedundantParentheses,PyRedundantParentheses,PyRedundantParentheses
+class TestTaskDetails():
+
+    def test_that_person_can_view_task_details(self, base_url, selenium):
+        # This test should be non-destructive, so it cannot log in to the site
+        home_page = HomePage(base_url, selenium).open()
+        assert not home_page.is_user_logged_in
+
+        # Click the first task in the Suggested Tasks list
```

Comment:
Done. I should have done that initially - They're from the one and done issue text and I used them to make sure I wrote the test accurately but they're basically verbatim what the code stays.